src/libraries/Microsoft.PowerFx.Core/Public/IPowerFxScope.cs

```diff
@@ -21,5 +21,12 @@ public interface IPowerFxScope
         /// Provide intellisense for expression
         /// </summary>
         IIntellisenseResult Suggest(string expression, int cursorPosition);
+
+        /// <summary>
+        /// Provider quick fix suggesions.
+        /// </summary>
+        /// <param name="expression">The formula expression.</param>
+        /// <returns>Collection of quick fixes.</returns>
+        QuickFixResult[] QuickFix(string expression);
```

Comment:
>QuickFixResult [](http://example.com/codeflow?start=8&length=14)

This looks like it populates the menu of available fixes, but once they select one of the quick fixes, how does it convey the actual text update to apply?

Reply:
This is executed by via code action command (monaco editor).